packages/core/package.json

```diff
@@ -36,32 +36,21 @@
     "backbone-undo": "0.2.6",
     "codemirror": "5.63.0",
     "codemirror-formatting": "1.0.0",
+    "html-entities": "~1.4.0",
```

Comment:
@artf this is needed to stop error: 

* https://github.com/vuejs/vuepress/issues/2996

Looks like we should keep it.